Caboodle.ruleset

```diff
@@ -0,0 +1,210 @@
+    <Rule Id="SA1501" Action="Warning" />
+    <Rule Id="SA1502" Action="Warning" />
+    <Rule Id="SA1503" Action="None" />
+    <Rule Id="SA1504" Action="Warning" />
+    <Rule Id="SA1505" Action="Warning" />
+    <Rule Id="SA1506" Action="Warning" />
+    <Rule Id="SA1507" Action="Warning" /><!-- TODO: verify SA1507CodeMustNotContainMultipleBlankLinesInARow -->
+    <Rule Id="SA1508" Action="Warning" />
+    <Rule Id="SA1509" Action="Warning" />
+    <Rule Id="SA1510" Action="Warning" />
+    <Rule Id="SA1511" Action="Warning" />
+    <Rule Id="SA1512" Action="None" />
+    <Rule Id="SA1513" Action="Warning" />
+    <Rule Id="SA1514" Action="Warning" />
+    <Rule Id="SA1515" Action="Warning" />
+    <Rule Id="SA1516" Action="Warning" />
+    <Rule Id="SA1517" Action="Warning" />
+    <Rule Id="SA1518" Action="Warning" />
+    <Rule Id="SA1519" Action="Warning" />
+    <Rule Id="SA1520" Action="Warning" />
+    <Rule Id="SA1600" Action="None" />
+    <Rule Id="SA1601" Action="None" />
+    <Rule Id="SA1602" Action="None" />
+    <Rule Id="SA1603" Action="None" />
+    <Rule Id="SA1604" Action="None" />
+    <Rule Id="SA1605" Action="None" />
+    <Rule Id="SA1606" Action="None" />
+    <Rule Id="SA1607" Action="None" />
+    <Rule Id="SA1608" Action="None" />
+    <Rule Id="SA1609" Action="None" />
+    <Rule Id="SA1610" Action="None" />
+    <Rule Id="SA1611" Action="None" />
+    <Rule Id="SA1612" Action="None" />
+    <Rule Id="SA1613" Action="None" />
+    <Rule Id="SA1614" Action="None" />
+    <Rule Id="SA1615" Action="None" />
+    <Rule Id="SA1616" Action="None" />
+    <Rule Id="SA1617" Action="None" />
+    <Rule Id="SA1618" Action="None" />
+    <Rule Id="SA1619" Action="None" />
+    <Rule Id="SA1620" Action="None" />
+    <Rule Id="SA1621" Action="None" />
+    <Rule Id="SA1622" Action="None" />
+    <Rule Id="SA1623" Action="None" />
+    <Rule Id="SA1624" Action="None" />
+    <Rule Id="SA1625" Action="None" />
+    <Rule Id="SA1626" Action="Warning" />
+    <Rule Id="SA1627" Action="None" />
+    <Rule Id="SA1628" Action="None" />
+    <Rule Id="SA1629" Action="None" />
+    <Rule Id="SA1630" Action="None" />
+    <Rule Id="SA1631" Action="None" />
+    <Rule Id="SA1632" Action="None" />
+    <Rule Id="SA1633" Action="None" />
+    <Rule Id="SA1634" Action="None" />
+    <Rule Id="SA1635" Action="None" />
+    <Rule Id="SA1636" Action="None" />
+    <Rule Id="SA1637" Action="None" />
+    <Rule Id="SA1638" Action="None" />
+    <Rule Id="SA1639" Action="None" />
+    <Rule Id="SA1640" Action="None" />
+    <Rule Id="SA1641" Action="None" />
+    <Rule Id="SA1642" Action="None" />
+    <Rule Id="SA1643" Action="None" />
+    <Rule Id="SA1644" Action="None" />
+    <Rule Id="SA1645" Action="None" />
+    <Rule Id="SA1646" Action="None" />
+    <Rule Id="SA1647" Action="None" />
+    <Rule Id="SA1648" Action="None" />
+    <Rule Id="SA1649" Action="Warning" /><!-- TODO: verify SA1649FileNameMustMatchTypeName -->
```

Comment:
Must out class names and file names match?

Reply:
No